### DISCUSSION TITLE
Line break in embedded chat

### DISCUSSION THREAD

**Discussion by pascalvallet** [FlowiseAI/Flowise#1425] [Q&A]
Forgive my naive question; I have designed my first app with Flowise. I am a school principal, and I have developed an AI bot to help teachers review their report card comments to align them with the guidelines. I have been using Flowise for the last 2 hours. The results are wonderful within the Flowise app. As an admin, the chatbot is working very well. However, I need to embed the chat within an HTML page on a Google site, and the result is very different. The quality of the output is great, but the answers have many empty lines. Every time the bot goes to a new line for a sentence, it adds 2 empty lines, and when the bot would add an empty line to separate two paragraphs, there are 4 empty lines. 

How can I make sure that the output does not add extra empty lines?

**Comment by HenryHengZJ** [comment]
We have added a [PR](https://github.com/FlowiseAI/FlowiseChatEmbed/pull/75) to properly format the message on chat embedded. Will update again once its merged and released

**Reply by HenryHengZJ** [reply]
https://github.com/FlowiseAI/FlowiseChatEmbed/pull/75

**Comment by pascalvallet** [comment]
great thanks

*Pascal Vallet*
Junior School Principal
E: ***@***.***
P: +1 212-584-3061 <+1%20212-584-3061>
W: http://www.unis.org
[image: Facebook] <https://facebook.com/unisny>[image: LinkedIn]
<https://www.linkedin.com/in/pvallet>[image: Twitter]
<https://twitter.com/unisnyc>[image: Instagram]
<https://www.instagram.com/unisny>
